test/functional/page_objects/visualize_page.js

```diff
@@ -776,6 +776,13 @@ export function VisualizePageProvider({ getService, getPageObjects }) {
       .getAttribute('height');
     }
 
+    getDataTableHeaders() {
+      return remote
+      .setFindTimeout(defaultFindTimeout * 2)
+      .findByCssSelector('table.table.table-condensed thead')
+      .getVisibleText();
+    }
```

Comment:
can you make this:
```
async getDataTableHeaders() {
  const dataTableHeader = await retry.try(async () => await find.byCssSelector('table.table.table-condensed thead'));
  return await dataTableHeader.getVisibleText();
}
```

I'm in the process of converting most of the functions in this file over to that style.